pkgs/development/tools/java/jprofiler/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, makeDesktopItem
+, undmg
+, jdk }:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  pname = "jprofiler";
+
+  # 11.1.4 is the last version which can be unpacked by undmg
+  # See: https://github.com/matthewbauer/undmg/issues/9
+  version = if stdenv.isLinux then "13.0.2" else "11.1.4";
+  nameApp = "JProfiler";
+
+  meta = with lib; {
+    description = "JProfiler's intuitive UI helps you resolve performance bottlenecks";
+    longDescription = ''
+      JProfiler's intuitive UI helps you resolve performance bottlenecks,
+      pin down memory leaks and understand threading issues.
+    '';
+    homepage = "https://www.ej-technologies.com/products/jprofiler/overview.html";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ catap ];
+  };
+
+  src = fetchurl (if stdenv.isLinux then {
```

Comment:
Please make the entire fetcher conditionnal, not only the arguments to fetchurl.